src/engine/QueryPlanner.cpp

```diff
@@ -952,6 +954,49 @@ ParsedQuery::GraphPattern QueryPlanner::seedFromInverse(
   return seedFromPropertyPath(right, path.children_[0], left);
 }
 
+// _____________________________________________________________________________
+ParsedQuery::GraphPattern QueryPlanner::seedFromNegated(
+    const TripleComponent& left, const PropertyPath& path,
+    const TripleComponent& right) {
+  AD_CORRECTNESS_CHECK(!path.children_.empty());
+  Variable forwardVar = generateUniqueVarName();
+  Variable inverseVar = generateUniqueVarName();
+  ParsedQuery::GraphPattern forwardPattern =
+      seedFromIri(left, PropertyPath::fromVariable(forwardVar), right);
+  ParsedQuery::GraphPattern inversePattern =
+      seedFromIri(right, PropertyPath::fromVariable(inverseVar), left);
```

Comment:
Add a comment on what this does (collect all the inverse and not inverse paths, and transfer them to a UNION + apply filters, and then state an example here (so that one doesn't have to jump to the QueryPlanner tests.